doc/installation.rst

```diff
@@ -13,7 +15,63 @@ or::
 
     pip install ts-salobj[==version]
 
-Configure Kafka
-===============
 
-Set environment variables ``LSST_KAFKA_BROKER_ADDR`` and ``LSST_SCHEMA_REGISTRY_URL``.
+Run a Local Kafka Server
+========================
+
+The most convenient way to develop salobj-based software is to run a local Kafka server, as follows:
+
+* If running salobj code in a Docker image:
+
+  * Make sure Docker has a network named ``ts_salobj_default``::
+
+      docker network ls
+
+    If not, create it with::
+
+      docker network create ts_salobj_default
+
+  * Run your Docker image with this option, to make this network available::
+
+    --network=ts_salobj_default
+
+  * Leave environment variables ``LSST_KAFKA_BROKER_ADDR`` and ``LSST_SCHEMA_REGISTRY_URL`` undefined.
+    The default values assume you are running salobj code inside a docker image.
+
+* If running salobj code directly on your computer, instead of inside a Docker image:
+
+  * Set the following environment variables::
+
+      export LSST_KAFKA_BROKER_ADDR="127.0.0.1:9092"
+      export LSST_SCHEMA_REGISTRY_URL="http://schema-registry:8081"
+
+* To start and stop Kafka, use the following bash/zsh functions::
+
+    function kafka_run() {
+        pushd /Users/rowen/UW/LSST/tsrepos/ts_salobj
```

Comment:
Maybe add a stub here for the path? Something like:

```
        pushd <tsrepos-path>/ts_salobj
```

and say that one must replace `<tsrepos-path>` with the path to their directory with ts_salobj repository?

Reply:
Good catch. Thank you!